scripts/lib/common.sh

```diff
@@ -28,3 +28,27 @@ function display_timelines() {
     echo "TIMELINE: Conformance tests took $CONFORMANCE_DURATION seconds."
     echo "TIMELINE: Down processes took $DOWN_DURATION seconds."
 }
+
+function run_warm_ip_test() {
+    $KUBECTL_PATH set env ds aws-node -n kube-system WARM_IP_TARGET=2
+    $KUBECTL_PATH set env ds aws-node -n kube-system MINIMUM_IP_TARGET=10
+    #Sleep a couple seconds to ensure propogation
+    sleep 2
```

Comment:
Nit: "propagation"

Also, setting these requires the `aws-node` pods to restart, and that probably takes up to 20seconds per node, meaning we need to wait for 1 minute or more.